cmd/tsdb/main.go

```diff
@@ -473,6 +474,7 @@ func analyzeBlock(b *tsdb.Block, limit int) {
 			labelsUncovered[lbl.Name] += uncovered
 			labelpairsUncovered[key] += uncovered
 			labelpairsCount[key]++
+			labelValueLengths[lbl.Name] += uint64(len(lbl.Value))
```

Comment:
This is going to over-count, you only want to this once per value. Look at how highest cardinality is done.

Reply:
:+1: Let me know if the new implementation looks better!